docs/advanced/auto-populate-mutations.md

```diff
@@ -43,6 +43,9 @@ own. It should also be placed after the `dedupExchange` to avoid unnecessary wor
 
 Adding the `populateExchange` now enables us to use the `@populate` directive in our mutations.
 
+The `schema` option is the introspection result for your backend graphql schema, more information
+about how to get your schema can be found [here](../graphcache/schema-awareness.md#getting-your-schema).
```

Comment:
Sorry for the nit, but let’s expand this to “can be found [in the “Schema Awareness” Page of the Graphcache documentation.]